src/Tests/SettingsTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
```

Comment:
Perhaps a different test class, but I would also expect a test that the thumbnail configuration form is attached to the node types edit forms - like admin/structure/types/manage/article.  I see the unit tests that handle testing the form itself, but this would be to make sure it shows where expected.